### DISCUSSION TITLE
Data processing operations

### DISCUSSION THREAD

**Discussion by azntaiji** [teableio/teable#736] [Ideas]
Feature similar to SeaTable's:

[SeaTable data processing operations](https://seatable.io/en/docs/datenverarbeitung/datenverarbeitungsoperationen-anlegen/)

**Comment by tea-artist** [comment]
It seems there isn't much difference from what the formula field does?

**Reply by azntaiji** [reply]
No it's quite different from formulas. This is more post processing / automation based. 

Example: I can automatically do a vlookup type of operation between two tables to link records without having to do so manually.

**Reply by tea-artist** [reply]
This capability should be fully covered by the Automation feature, which is on the list of upcoming functionalities.
